client/lib/posts/post-edit-store.js

```diff
@@ -294,7 +294,8 @@ function dispatcherCallback( payload ) {
 			_isAutosaving = false;
 			if ( ! action.error ) {
 				_previewUrl = utils.getPreviewURL(
-					assign( { preview_URL: action.autosave.preview_URL }, _savedPost )
+					assign( { preview_URL: action.autosave.preview_URL }, _savedPost ),
```

Comment:
This is likely not relevant to this PR, but we could drop the `_.assign` and replace it with a spread:
`{ preview_URL: action.autosave.preview_URL, ..._savedPost }`